### PR TITLE
Count of string p in string s #1086

### PR DESCRIPTION
**Pull request by sukritidawar**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added the solution in cp folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by sukritidawar** [comment]
@Kajol-Kumari please review

**Comment by sukritidawar** [comment]
@HarshCasper @plazzy99 please review

File: C-Plus-Plus/cp/CountofPinS.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        return check_p_in_s(p, s, stri + 1, count + 1);
+    }
+    else{ //When string p does not match the substring in s, we just move to the next character without increasing the count
+        return check_p_in_s(p, s, stri + 1, count); 
+    }
+}
+
+int main(){
+    string p, s;
+    cin >> p;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/CountofPinS.cpp]: Display message regarding what you are taking input and showing output

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@sukritidawar Do the suggested changes

File: C-Plus-Plus/cp/CountofPinS.cpp (hunk 1 above)
Comment (sukritidawar) [reply C-Plus-Plus/cp/CountofPinS.cpp]: Done

**Review by siddharth25pandey — APPROVED** [review]
@sukritidawar Good to go
Able to merge
Thanks!

**Comment by sukritidawar** [comment]
@HarshCasper please review

**Review by Kajol-Kumari — APPROVED** [review]
Can we please keep only the required changes in readme and remove the rest? Rest looking good to me!

**Comment by sukritidawar** [comment]
> Can we please keep only the required changes in readme and remove the rest? Rest looking good to me!

I didn't get what actually do I have to change in readme...

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**